contrib/cirrus/runner.sh

```diff
@@ -199,8 +199,16 @@ function _run_altbuild() {
 }
 
 function _run_release() {
-    if bin/podman info |& grep -Eq -- '-dev'; then
-        die "Releases must never contain '-dev' in output of 'podman info'"
+    # TODO: These tests should come from code external to the podman repo.
+    # to allow test-changes (and re-runs) in the case of a correctible test
+    # flaw or flake at release tag-push time.  For now, the test is here
+    # given it's simlicity.
+
+    msg "Checking podman release (or potential release) criteria."
+    info_output=$(bin/podman info 2>&1)
+    if grep -Eq -- '-dev'<<<"$info_output"; then
+        die "Found unexpected '-dev' in output of 'podman info':
```

Comment:
Could you leave my patch as it was? `podman info` is an enormous long string, and it is anti-helpful to include the whole thing. The reader only wants/needs to see the `-dev` strings.

BTW I deliberately removed the `-E` in my recommended patch. `-E` is not needed here, and its presence confuses the reader.

Reply:
I'm in favor of the more verbose output (unprocessed by `grep`) since (for now) it supports a human catching other release-affecting mistakes, like a wrong/unexpected buildah or conmon version.  At least until we get more/improved release tests in place.

Reply:
I'm afraid I disagree. I cannot imagine your scenario happening in reality: 

Reader: "oh, look, a failure, let me look at the entire output, cross-reference it against my expected versions, oh, wow, an unexpected difference in conmon! Thank goodness for full output".

 What I think is more likely:

Reader: "oh look, a failure, what is the error? Do I actually have to look at ONE HUNDRED AND TWO LINES of crap just to see what the problem is? Okay, I'll use my browser search function, oh darn I used two dashes for --dev, that doesn't find anything, okay let's try again, oh there it is, all right I'll fix it"
(voiceover: what our debugger doesn't know is that there were *two* -devs in the output).

Please just show the relevant error output and only the relevant error output. No distractions please.

Reply:
I think you missed this (it's been a long week). It is not resolved. Please resolve it.

Reply:
I hear what you're saying, but I think we're spending too much time on a relatively minor item.  I'm afraid of making your changes costing a lot of testing time.  In other words, I would need an additional run to verify (fake) `$CIRRUS_TAG` activation works (fails), another run to verify the manual-activation works (fails), then a third run to have a clean CI result for merging.  This is assuming there aren't any actual problems found that need fixing  :confounded: 

IMHO, it's simply not worth our time for a code-path that is used what...three-five times per year, maybe?   I'd rather spend time working improving the overall podman-release experience, and the myriad of other testing issues which need more urgent attention.

Reply:
Then just remove the changes to runner.sh and resubmit. I'm not going to approve this as it is, it's made everything worse. I'll just fix it myself later.

Reply:
What motivation have you provided, for me to approve a re-fix?

 I think for the greater good and grander-schemes...it's past time we agree to disagree.  Our foreheads will thank us for it :smiley:

Reply:
The fact that I provided a simple, clean, tested fix... which you ignored in favor of a broken one. Come on, this is very simple. Please either use my fix to runner.sh (which, again, I tested fairly extensively) or undo your changes to runner.sh. I provided my reasoning above: dumping the entire `podman info` output is cruel and harmful.

Reply:
Ed, we're simply not going to agree on the verbosity level.  But we also need to rise above perfectionism and pettiness, it doesn't help anybody. Since any changes will require re-testing anyway...and this test isn't a long-term solution...here's a compromise:

* I'll remove this automated check (here) and instead include it as a manual check in the release docs (short-term fix).
* I've got a jira card open already, as a "SPIKE" to fully spec. requirements for improved release-related automation and manual processes.  I'd like your assistance completing this (helping define the spec).
* I'd like you to take on the (assumed) followup card for adding (proper) automated release tests (long-term fix).

okay?